src/Fusonic/Linq/Linq.php

```diff
@@ -134,8 +139,14 @@ public function take($count)
         if ($count == 0) {
             return new Linq([]);
         }
+        $innerIterator = $this->iterator;
+        if ($innerIterator instanceof \Iterator === false) {
```

Comment:
Please use `if (!($innerIterator instanceof \Iterator))` so that we use a consistent coding style (see e.x. https://github.com/fusonic/linq/blob/master/src/Fusonic/Linq/Linq.php#L330)